scripts/analyze_all_repos_github_actions.py

```
@@ -0,0 +1,434 @@
+    ) -> list[dict[str, Any]]:
+        """Fetch workflow runs for a specific repository.
+
+        Args:
+            owner: Repository owner
+            repo: Repository name
+            days: Number of days to look back
+
+        Returns:
+            List of workflow run objects
+        """
+        created_after = (datetime.now() - timedelta(days=days)).isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)